react-native-app/views/SignUpView.tsx

```diff
@@ -165,6 +250,12 @@ const styles = StyleSheet.create({
     flex: 5,
     marginTop: 20,
     marginBottom: 10
+  },
+  validation: {
+    color: 'red', 
```

Comment:
Same here, about making sure this 'red' is the same red used everywhere else...or if this is the classic "you forgot to input these required fields" kind of red then it's fine.

Reply:
Updated to use the same as used throughout the app. It is noticeable and simple to understand as a user.